mscore/pagesettings.ui

```diff
@@ -151,8 +147,19 @@
           </property>
          </widget>
         </item>
-        <item row="0" column="0" colspan="2">
-         <widget class="QComboBox" name="pageGroup"/>
+        <item row="3" column="0">
+         <widget class="QRadioButton" name="portraitButton">
+          <property name="text">
+           <string>portrait</string>
```

Comment:
I wanted to ask whether you want me to change "portraitButton" to "PortraitButton" as well ?
Currently I am thinking of changing "<string>portrait</string>"to <string>Portrait</string>.